comp/core/tagger/impl/local_tagger_test.go

```diff
@@ -6,9 +6,9 @@
 package taggerimpl
 
 import (
-	"context"
```

Comment:
Same question as before to make sure I'm not missing anything.
Is there something new on this file or you just copied things that we had in `impl/tagger_test.go` before?

Reply:
Nope, just a copy.